### DISCUSSION TITLE
BrickStore not Mass Updating properly

### DISCUSSION THREAD

**Discussion by happydaybricks** [rgriebl/brickstore#923] [Q&A]
![19d05533-7f1f-48f5-9528-e854d1f16576](https://github.com/user-attachments/assets/84ff2730-68a0-4520-9eee-76a2ce11a975)
![da75eb36-954d-4174-ab65-e19fb3806136](https://github.com/user-attachments/assets/5a90fdd3-03d1-434e-839c-458b5396a8bc)

Hey guys, so I've been using BrickStore for alittle bit now and I've been running into issues where the Mass Update doesn't update properly. In the past, I would just go into BrickLink and update it myself instead of trying to figure out why it was happening. This time around I'm updating my inventory just making sure everything is up to date before adding a bunch of sets to it. I had to update 4 lines. 

- Line 1: inventory changed from 18 to 16 (the export is showing 18 and adding 1)
- Line 2: inventory did not change but the remark did (no update in export)
- Line 3: inventory changed from 6 to 0 (no update in export)
- Line 4: inventory changed from 16 to 10 (no update in export)

I also feel like this is why my inventory is off because the updates don't go through properly when I update from BrickOwl on BrickStore. (I'm looking at BrickFreedom going forward after this).

Can anyone give me some information on what I could possibly be doing wrong?

**Comment by jay-marks** [comment]
Hi,
I know this was a while back but noted there'd been no reply.
If you (doubtful) have the XML then this is where we'd need to look - BrickLink works on deltas from what you submit and what is current status - and is based on LOT IDs rather than Item IDs. Maybe check if you have some duplicates by running a consolidate in BrickStore on a COPY of current inventory, see if there are any.
If you are still having problems am happy to review the XML output and see how this might deviate from what is expected (it's BrickLink that does the update so may be a challenge at their end if things aren't updating based on the XML)
Recommend paste the XML into your favourite text editor and save with appropriate name.
What do you use to manage sync between Link and Owl? BrickSync does offer some functionality to perform some updates.
